forked/naughty-words/ar

```diff
@@ -0,0 +1,38 @@
+سكس
```

Comment:
@manoharuss , for all these dictionary file, we can covert them into JSON. For example: `ar -> ar.json` and then put all these words in an array.

```
[
  طيز,
  شرج,
  لعق,
  لحس,
  مص
]
```

This helps in easy usability of them in the detector and can be be applied with all the array operations.

Reply:
I noticed this pr actually imported the files, rather than using the npm module that @bkowshik added to the package.json in  2017 https://github.com/mapbox/osm-compare/pull/141